WcaOnRails/app/views/posts/_post.html.erb

```diff
@@ -22,6 +22,6 @@
     <div>
       <em>Announced by <%= post.author.name %> on <%= local_time(post.created_at) %></em>
     </div>
-    <%= post.body.html_safe %>
```

Comment:
Haha, wow. That's funny. This example code looks kind of schizophrenic: http://guides.rubyonrails.org/i18n.html#adding-date-time-formats. Leaving this as is seems fine to me.